psydac/feec/global_projectors.py

```diff
@@ -275,11 +275,11 @@ def __init__(self, L2, quads=None):
         slices     = tuple(slice(p+1,-p-1) for p in L2.degree)
 
         if len(self.D) == 1:
-            self.func = evaluate_dof_3form_1d
+            self.func = evaluate_dofs_1d_1form
         elif len(self.D) == 2:
-            self.func = evaluate_dof_3form_2d
+            self.func = evaluate_dofs_2d_2form
         elif len(self.D) == 3:
-            self.func = evaluate_dof_3form_3d
+            self.func = evaluate_dofs_3d_3form
         else:
             raise ValueError('H1 projector of dimension {} not available'.format(str(len(self.N))))
 
```

Comment:
on L290 it should not be 'space V1' as the index for an L2 projection is n = dimension
(same comment for "1-form" on L295)